CRM/Upgrade/Incremental/php/FiveFiftyFour.php

```diff
@@ -21,6 +21,15 @@
  */
 class CRM_Upgrade_Incremental_php_FiveFiftyFour extends CRM_Upgrade_Incremental_Base {
 
+  public function setPreUpgradeMessage(&$preUpgradeMessage, $rev, $currentVer = NULL) {
+    parent::setPreUpgradeMessage($preUpgradeMessage, $rev, $currentVer);
+    if ($rev === '5.54.alpha1') {
+      if (\Civi::settings()->get('civicaseActivityRevisions')) {
+        $preUpgradeMessage .= '<p>' . ts('The setting at Administer - CiviCase - CiviCase Settings for <em>Enable deprecated Embedded Activity Revisions</em> is enabled.') . '<ul><li>' . ts('While revisions created prior to 5.54 will remain in the database, the setting will now be ignored and new revisions will no longer be created.') . '</li></ul></p>';
```

Comment:
> The setting at Administer - CiviCase - CiviCase Settings for

I would strongly encourage some demarcation (like `"..."` or `<em>...</em>`) around the navigational path. Additionally, I searched to see what path-delimiters are used, and `>` (along with close variants  `>>` and `&raquo;`) are much more common.

eg

```diff
-Administer - CiviCase - CiviCase Settings
+"Administer > CiviCase > CiviCase Settings"
```

> the setting will now be ignored and new revisions will no longer be created.

Perhaps remind the reader that they can enable general DB logging?

Reply:
Thanks have updated. (I would be super-curious to have it collect stats on how many people have this setting enabled given it's been broken in the UI for years, but it's more fun to make up my own stats anyway.)

<img width="521" alt="Untitled2" src="https://user-images.githubusercontent.com/2967821/184720711-f1486223-b18f-4460-ac2a-92e8d75b25a5.png">

Reply:
It might be possible to gain insight into those stats by checking EBay listings generated by https://lab.civicrm.org/extensions/caseactivityrevisionmigrator

But also, it might be good to include a link to that extension from here :)

Reply:
Haha! Or maybe remove the whole bullet and instead link to the lab ticket or a whittled-down lab snippet.
And I should update the ext readme to make it clear NO REFUNDS.